docs/api.md

```diff
@@ -480,10 +479,7 @@ This class defines a workflow for building images
 - plugins_errors
 - plugins_timestamps
 - postbuild_plugins_conf
-- postbuild_results
 - prebuild_plugins_conf
-- prebuild_results
-- prepub_results
```

Comment:
should add `plugin_results`  here as well as `plugins_conf` and remove the rest of `*_plugins_conf` so that we don't have to clean up later

Reply:
IMO, The section "##### DockerBuildWorkflow instance variables" should be rewritten at all, . IIRC, we are going to update various documentation for OSBS2, and this one should be part of the that.